lib/matplotlib/tests/test_usetex.py

```diff
@@ -80,6 +81,23 @@ def test_minus_no_descent(fontsize):
     assert len({*heights.values()}) == 1
 
 
+@pytest.mark.skipif(_has_tex_package('xcolor.sty'),
```

Comment:
this should be inverted? (`not _has_tex_package`)

Reply:
Quite right, but also, it should not have `.sty` either.